crates/kornia-image/src/image.rs

```diff
@@ -52,14 +52,20 @@ impl From<ImageSize> for [u32; 2] {
     }
 }
 
-#[derive(Clone)]
-/// Represents an image with pixel data.
+/// A representation of an image with pixel data.
 ///
-/// The image is represented as a 3D Tensor with shape (H, W, C), where H is the height of the image,
-pub struct Image<T, const C: usize>(pub Tensor3<T, CpuAllocator>);
+/// The image is stored as a three-dimensional tensor with the shape
+/// `[height, width, channels]`.
+pub struct Image<T: Clone, const C: usize>(pub Tensor3<T, CpuAllocator>);
```

Comment:
this should happen to the tensor struct, and it's a simple as you agent is telling you